pkg/apis/events/v1beta1/conversion.go

```diff
@@ -56,3 +64,31 @@ func Convert_core_Event_To_v1beta1_Event(in *k8s_api.Event, out *v1beta1.Event,
 	out.DeprecatedCount = in.Count
 	return nil
 }
+
+func AddFieldLabelConversionsForEvent(scheme *runtime.Scheme) error {
+	return scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("Event"),
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "regarding.kind":
+				return "involvedObject.kind", value, nil
+			case "regarding.namespace":
+				return "involvedObject.namespace", value, nil
+			case "regarding.name":
+				return "involvedObject.name", value, nil
+			case "regarding.uid":
+				return "involvedObject.uid", value, nil
+			case "regarding.apiVersion":
+				return "involvedObject.apiVersion", value, nil
+			case "regarding.resourceVersion":
+				return "involvedObject.resourceVersion", value, nil
+			case "regarding.fieldPath":
+				return "involvedObject.fieldPath", value, nil
+			case "reportingController":
```

Comment:
`reportingController` is always set  with `Component` so this is fine, but `ReportingInstance` Exists in core too and v1beta1 we might want to add it